src/screens/LiquidityPools/AddLiquidity.js

```diff
@@ -122,27 +122,24 @@ const AddLiquidityScreen = ({
   useEffect(() => {
     resetEstimateTransaction();
   }, []);
+
   const [assetsValues, setAssetsValues] = useState(['', '']);
-  const [poolTokenAmount, setPoolTokenAmount] = useState('0');
   const [fieldsValid, setFieldsValid] = useState([true, true]);
+  const [poolTokenAmount, setPoolTokenAmount] = useState('0');
 
   const { pool } = navigation.state.params;
   const poolStats = getPoolStats(pool, liquidityPoolsReducer);
 
   const tokensData = pool.tokensProportions.map(({ symbol }) => supportedAssets.find(asset => asset.symbol === symbol));
   const poolTokenData = findSupportedAsset(supportedAssets, pool.uniswapPairAddress);
 
+  const [debouncedAssetsValues] = useDebounce(assetsValues, 500);
   useEffect(() => {
-    if (!assetsValues.every(f => !!parseFloat(f)) || !fieldsValid.every(f => f)) return;
-    const erc20Token = tokensData[1];
-    if (!erc20Token) return;
-    calculateAddLiquidityTransactionEstimate(
-      pool,
-      assetsValues,
-      poolTokenAmount,
-      tokensData,
-    );
-  }, [assetsValues, fieldsValid]);
+    if (!tokensData.every((token) => token != null)) return;
+    if (!assetsValues.every((f) => !!parseFloat(f))) return;
+    if (!fieldsValid.every((f) => f)) return;
```

Comment:
Can we squeeze these into one `if` statement? Also, `!=` => `!==`

Reply:
Re `!=` operator: `!= null` is equal to `!== null || !== undefined`, so that's intended.

Reply:
I've actually thought that separate simple `if(xxx) return`s where easier to read, but can change them to `if (xxx || yyy || zzz) return`, if you find it other way around.